.github/README.md

```diff
@@ -245,7 +247,7 @@ manual ──────────► └────────────
   - Build the code pushed on a default image. (`node` image from docker hub).
   - Run the tests against the build.
 * Workflow confirms code is not "broken".
-* Manual trigger supported. Enables to select image from docker hub or local files (e.g. `erbium-buster-slim`) to build code on.
+* Manual trigger supported. Enables to node version.
```

Comment:
Nit: should be "Enables selecting the node version"?

Reply:
Fixed.